package.json

```diff
@@ -11,15 +11,15 @@
   },
   "dependencies": {
     "@gravity-ui/axios-wrapper": "^1.3.0",
-    "@gravity-ui/chartkit": "^4.15.0",
-    "@gravity-ui/components": "^2.9.1",
-    "@gravity-ui/date-utils": "^1.1.1",
-    "@gravity-ui/i18n": "^1.0.0",
+    "@gravity-ui/chartkit": "^4.20.1",
+    "@gravity-ui/components": "^2.12.0",
+    "@gravity-ui/date-utils": "^2.1.0",
```

Comment:
charkit@4 depends on date-utils@1 https://github.com/gravity-ui/chartkit/blob/v4/package.json#L50

Reply:
Reduced its version. Will be updated together with uikit6